src/components/common/RadioGroup.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <span>{{ groupLabel }}</span>
+  <div class="flex justify-between items-center">
+    <div v-for="option in options" :key="option.key">
```

Comment:
W sumie v-for możesz wrzucić w label, a także każdy atrybut w nowej linii i kolejność atrybutów